doc/source/rllib/package_ref/replay-buffers.rst

```diff
@@ -0,0 +1,57 @@
+.. _replay-buffer-reference-docs:
+
+ReplayBuffer API
+================
+
+These classes don't take into account the separation of experiences from different policies, which we use in multi-agent settings.
+
+ray.rllib.utils.replay_buffers.replay_buffer
+---------------------------------------------
+
+.. autoclass:: ray.rllib.utils.replay_buffers.replay_buffer.StorageUnit
+    :members:
+
+.. autoclass:: ray.rllib.utils.replay_buffers.replay_buffer.ReplayBuffer
+    :members:
+    :show-inheritance:
+
+ray.rllib.utils.replay_buffers.prioritized_replay_buffer
+--------------------------------------------------------
+
+.. autoclass:: ray.rllib.utils.replay_buffers.prioritized_replay_buffer.PrioritizedReplayBuffer
+    :members:
+    :show-inheritance:
+
+ray.rllib.utils.replay_buffers.reservoir_replay_buffer
+------------------------------------------------------
+
+.. autoclass:: ray.rllib.utils.replay_buffers.reservoir_replay_buffer.ReservoirReplayBuffer
+    :members:
+    :show-inheritance:
+
+
+MultiAgentReplayBuffer classes
+==============================
+
+These classes use the above, "single-agent", buffers as underlying buffers to facilitate splitting up experiences between policies.
```

Comment:
Maybe add one sentence on what multi-agent RL is: In multi-agent RL, more than one agent exists in the environment and not all of these agents may utilize the same policy (mapping M agents to N policies, where M <= N). This leads to the need for MultiAgentReplayBuffers that can store blabla

Also: "between the different agents' policies."